deployment/lib/lambdas/cf-url-rewriter.ts

```diff
@@ -0,0 +1,70 @@
+import { CloudFrontRequestEvent, CloudFrontRequest } from 'aws-lambda';
+
+const https = require('https');
+
+export async function handler(event: CloudFrontRequestEvent, context, callback): Promise<CloudFrontRequest> {
+  const request = event.Records[0].cf.request;
+
+  console.log('request', JSON.stringify(request));
```

Comment:
Does this log the request with all its data? Likely a bad idea with secrets. If you need to log something, just log `request.uri`?

Reply:
Let me remove this since it is originally just for debugging purpose.